dags/hivemind_website_ingestion.py

```
@@ -0,0 +1,98 @@
+        """
+        Getting all communities having discord from database
+        """
+        communities = ModulesWebsite().get_learning_platforms()
+        return communities
+
+    @task
+    def start_website_embedding(community_info: dict[str, str | datetime | list]):
+        load_dotenv()
+        community_id = community_info["community_id"]
+        platform_id = community_info["platform_id"]
+        links = community_info["urls"]
```

Ruff

Local variable `links` is assigned to but never used

Remove assignment to unused variable `links`

(F841)